### ISSUE TITLE
is there a way to disallow all apps to post on Main feed. YES NO to post on feed or when we set to private

### ISSUE THREAD

**Issue by spirogg** [PHPfox-Official/phpfox-v4-feature-requests#717]
Hi I've noticed not all apps have a feature to turn off posting to Main Feed
some have allow posting on Main Feed YES NO 
but I would like to have the option to Disallow posting on main feed for all apps.

Also when we set to private example: ( forums set a forum to private then it will not post to the feed.)
but the forums that are not private can post to the feed ? 

same as Videos - when upload a private video it do not post on main feed.

so basically:
**when any app that has private feature and we select private - it will not post on Main Feed**

and also just a YES NO option for each app to post or not post on Main Feed

thanks

Spiro

**Comment by spirogg** [comment]
Thank you  !!!!! :)

**Comment by david-phpFox** [comment]
We implemented it in the newest version.
